secretmanager/snippets/regional_samples/add_regional_secret_version.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for adding a secret version to a
+regional secret with the specified payload to an existing regional secret.
+"""
+
+import argparse
+
+# [START secretmanager_v1_add_regional_secret_version]
+from google.cloud import secretmanager_v1
+import google_crc32c  # type: ignore
+
+
+def add_regional_secret_version(
+    project_id: str, location_id: str, secret_id: str, payload: str
+) -> secretmanager_v1.SecretVersion:
+    """
+    Add a new secret version to the given secret with the provided payload.
+    """
```

Comment:
suggestion: can you provide a docstring explaining the arguments of this function?

Reply:
For the sake of consistency, and limited time, i will be adding the explanation of arguments in a different PR. As of now, they are in consistent with the current samples that are already live.